Clicker/ViewControllers/GroupControlsViewController.swift

```diff
@@ -238,3 +250,30 @@ extension GroupControlsViewController: AttendanceViewControllerDelegate {
     }
 
 }
+
+// MARK: - PollSettingsSectionControllerDelegate
+extension GroupControlsViewController: PollSettingsSectionControllerDelegate {
+
+    func pollSettingsSectionControllerDidUpdate(_ sectionController: PollSettingsSectionController, to newValue: Bool) {
+        guard let pollsSettingModel = adapter.object(for: sectionController) as? PollsSettingModel else { return }
+        if pollsSettingModel.isEqual(toDiffableObject: filterSetting) {
```

Comment:
might be cleaner if you move this to the `guard let`

Reply:
eventually there's going to be more model toggles so each one would be in an if statement i think